storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.10.rst

```diff
@@ -0,0 +1,41 @@
+===========================
+|Percona XtraBackup| 2.3.10
+===========================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.10 on
+November 29th 2017. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.3.10/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3
+series.
+
+New Features
+============
+
+ |Percona XtraBackup| packages are now available for *Ubuntu 17.10 (Artful)*
+
+ |xbcrypt| has been rewritten to use the ``ds_decrypt`` library. This provides
```

Comment:
change with `ds_decrypt` is internal, we probably don't need to mention it. User-visible effect is that `xbcrypt` (not `xtrabackup` as stated) now supports `--encrypt-threads` option. Also please make sure to add `encrypt-threads` to `xbcrypt` option reference.